src/main/java/lesson02/ConsoleInput.java

```diff
@@ -0,0 +1,21 @@
+package lesson02;
+
+import java.util.Scanner;
+
+public class ConsoleInput {
+    static int inputNumberFromConsole (String textMessageForInput) {
+        Scanner scanner = new Scanner(System.in);
```

Comment:
A good point to cache this scanner object. (use the singleton pattern)

Reply:
If Singleton is used - which pros and cons it would give to you?
Are there any other alternatives?
P.S. do not spend much time on this tonight, we can discuss it during the meeting.

Reply:
Get some info of how to work with Singleton pattern. Pros and cons.